.circleci/config.yml

```diff
@@ -162,95 +164,94 @@ jobs:
 workflows:
   check-and-deploy-development:
     jobs:
-      - check-code-formatting:
-          context: api-nuget-token-context
+      - check-code-formatting
+      - build-and-test
+      - assume-role-development:
+          context: 
+            - api-assume-role-housing-development-context
+            - SonarCloud
+          requires:
+            - build-and-test
+          filters: 
+            branches:
+              only: develop
+      - terraform-init-and-apply-to-development:
+          requires:
+            - assume-role-development
+          filters:
+            branches:
+              only: develop
+      - migrate-database-development:
+          requires:
+            - terraform-init-and-apply-to-development
+          filters:
+            branches:
+              only: develop
+      - deploy-to-development:
+          requires:
+            - migrate-database-development
```

Comment:
I don't think this repository connects to any database directly, so it shouldn't have any code-first database code either, which could trigger any migrations.

Actually,... the migration job referenced within this workflow **doesn't seem to even exist** - the CircleCI should fail while parsing the `.circleci/config.yml` file.